env/.env.prod.db

```diff
@@ -0,0 +1 @@
+POSTGRES_PASSWORD=hydraDock3r!
```

Comment:
The filename implies that this is meant for a production environment. Given that: 

1. Why has this been added? 
2. If this is the prod value, is this now a leaked secret?

Reply:
1. You will need these files to seed the db
2. These were already in the repo in the env.dev.db so they aren't true production values
we can remove these before committing this MR, but they are here to get the app running.